### PR TITLE
Improve /search endpoint

### PR DESCRIPTION
**Pull request by Anakin100100**

### CODE REVIEW

File: db/migrate/20220212134157_add_area_fields_to_institution.rb [hunk 1]
```diff
@@ -1,7 +1,13 @@
 class AddAreaFieldsToInstitution < ActiveRecord::Migration[7.0]
   def change
+<<<<<<< HEAD
```
Comment (micorix) [inline db/migrate/20220212134157_add_area_fields_to_institution.rb:3]: Merge leftovers

File: db/schema.rb [hunk 2]
```diff
@@ -74,14 +74,23 @@
     t.integer "sport_facilities"
     t.float "working_time"
     t.float "students_per_teacher"
+<<<<<<< HEAD
```
Comment (micorix) [inline db/schema.rb:77]: Ditto

File: test.save [hunk 3]
```diff
@@ -0,0 +1,10 @@
+As a Main Backend Maintainer I:
```
Comment (micorix) [inline test.save:1]: Hi

1. don't we want to add it to the website?
2. OR maintainers.md / contributors.md? Why test.save?

File: app/controllers/institutions_controller.rb [hunk 4]
```diff
@@ -16,13 +16,14 @@ def index
     institutions = institutions.search_by_sports(@sports) unless @sports.nil?
```
Comment (micorix) [inline app/controllers/institutions_controller.rb:16]: How about improving fuzzy search?

**Review by micorix — CHANGES_REQUESTED** [review]

File: test.save (hunk 3 above)
Comment (Anakin100100) [reply test.save:1]: I didn't mean to include it there, I needed to copy it in a barebones editor to strip it from styling from word text editor. I was writing my cv yesterday.

File: app/controllers/institutions_controller.rb (hunk 4 above)
Comment (Anakin100100) [reply app/controllers/institutions_controller.rb:16]: For one sport the isn't much of a difference but fuzzy search doesn't work well with multiple different sports whereas the default full text search matches the record when any of the sports mentioned match.

Comment (micorix) [reply app/controllers/institutions_controller.rb:16]: Sry it was just a first line so i put there general comment. i should have been more clear, sry

**Review by micorix — APPROVED** [review]

**Pull request merged**